src/current/cockroachcloud/cloud-api.md

```diff
@@ -6,15 +6,17 @@ docs_area: manage
 cloud: true
 ---
 
-The Cloud API is a [REST interface](https://wikipedia.org/wiki/Representational_state_transfer) that allows you programmatic access to manage the lifecycle of clusters within your organization.
+The Cloud API is a [REST interface](https://wikipedia.org/wiki/Representational_state_transfer) that allows service accounts programmatic access to manage the lifecycle of clusters within your organization.
 
 ## API reference
 
-Refer to the [full API reference documentation](https://www.cockroachlabs.com/docs/api/cloud/v1) for detailed descriptions of the API endpoints and options.
+Refer to the [full API reference documentation](https://www.cockroachlabs.com/docs/api/cloud/v1.html#post-/api/v1/clusters) for detailed descriptions of the API endpoints and options.
 
 ## Call the API
 
-The API uses [bearer token authentication](https://swagger.io/docs/specification/authentication/bearer-authentication/), and each request requires a [secret key]({% link cockroachcloud/managing-access.md %}#api-access). The secret key is associated with a service account, and inherits the [permissions of the account]({% link cockroachcloud/managing-access.md %}#manage-service-accounts).
+The API uses [bearer token authentication](https://swagger.io/docs/specification/authentication/bearer-authentication/), and each request requires a [secret key]({% link cockroachcloud/managing-access.md %}#api-access), associated with a service account. The API calls you can make is limited by the roles granted to the service account you are using.
+
+Refer to: [Manage service accounts]({% link cockroachcloud/managing-access.md %}#manage-service-accounts).
```

Comment:
It would be great to include a sentence here to tell the user why they might want to go visit that link, rather than just push on through this page. (Similarly to what you've done on line 13.)